### ISSUE TITLE
SFC linting doesn't work with the header plugin

### ISSUE THREAD

**Issue by dietergeerts** [vuejs/eslint-plugin-vue#782]
**Tell us about your environment**	
* **ESLint version:**	 4.19.1
* **eslint-plugin-vue version:**	4.5.0
* **Node version:**	8.11.2

**Please show your full configuration:**

We have a multi-file setup, so it's hard to paste the complete setup here. 
We have the header plugin setup, and the plugin to lint Vue files

**What did you do?**	

```vue
<!--
  - Copyright 2019 Our company
  -->
<template>
...
```

**What did you expect to happen?**
That the linting error 'missing header' doesn't happen

**What actually happened?**

The linting error 'missing header' does happen

I think it will be because of the different parts of the SFC, no? I don't know if this is because of this plugin, or the header one, but as it only happens to our vue files, I suspect it's this one.

**Comment by dietergeerts** [comment]
https://github.com/Stuk/eslint-plugin-header
It's to ensure that our copyright header is correctly added.
I also saw this weekend that there is a similar plugin, so I could try that too, but I guess it will be the same problem. (https://github.com/nickdeis/eslint-plugin-notice)

**Comment by armano2** [comment]
this plugin expect that your copyrights are going to be within script tag 
```vue
<template> ... </template>
<script>
<!--
  - Copyright 2019 Our company
  -->
<script>

**Comment by ota-meshi** [comment]
As already commented, it seems that the `.vue` file and `eslint-plugin-header` are not compatible. You cannot use `eslint-plugin-header` with `.vue` file.
